### DISCUSSION TITLE
Unable to get files preview

### DISCUSSION THREAD

**Discussion by Cangio84** [globaleaks/globaleaks-whistleblowing-software#3624] [Q&A] [answered]
Recipient are unable to get preview of pdf attachment.
I used Chrome, Edge
Below a screenshot of the issue.

![file_type](https://github.com/globaleaks/GlobaLeaks/assets/144811052/cb62cb6e-ca9e-4f15-b4fc-b944409e9a01)
![Pdf_preview](https://github.com/globaleaks/GlobaLeaks/assets/144811052/973217bc-9576-4296-9d3e-e39507d57c66)

Thanks to all

**Comment by evilaliv3** [comment, marked as answer]
Thank you.

This has been reported today buy an other user. We will check and fix the issue within a bugfix release.

https://github.com/orgs/globaleaks/discussions/3624

**Reply by Cangio84** [reply]
It's a great sound.
Thank you so much

**Comment by Cangio84** [comment]
I updated to last version 4.13.9 and it work as well
Thank you so much
